eclipse-collections-code-generator/src/main/resources/test/map/mutable/abstractMutableObjectPrimitiveMapTestCase.stg

```diff
@@ -362,6 +362,21 @@ public abstract class AbstractMutableObject<name>MapTestCase extends AbstractObj
         Assert.assertEquals(Object<name>HashMap.newWithKeysValues(null, <(literal.(type))("50")>), map3);
     }
 
+    @Test
+    public void getAndPut()
+    {
+        MutableObject<name>Map\<Integer> map1 = this.getEmptyMap();
+        Assert.assertEquals(<(literal.(type))("200")>, map1.getAndPut(5, <(literal.(type))("100")>, <(literal.(type))("200")>)<delta.(type)>);
+        Assert.assertEquals(<(literal.(type))("100")>, map1.get(5)<delta.(type)>);
```

Comment:
Please explicitly box the keys here if they are `Integer` values.  Also, the values above 127 will be out of range for `ObjectByteHashMap`.

Reply:
fixed